lms/static/scripts/frontend_apps/errors.js

```diff
@@ -105,3 +118,46 @@ export function isLTILaunchServerError(error) {
     ].includes(error.errorCode)
   );
 }
+
+/**
+ * Format a user-facing message based on this error and optional contextual
+ * prefix, using the appropriate message information.
+ *
+ * @param {ErrorLike} error
+ * @param {String} [prefix]
```

Comment:
```suggestion
 * @param {string} [prefix]
```

Primitive types should be lower-case, as `typeof` would return.